system/include/webgpu/webgpu_cpp.h

```diff
@@ -1,139 +1,15 @@
 #ifndef WEBGPU_CPP_H_
 #define WEBGPU_CPP_H_
 
-#include "webgpu/webgpu.h"
-
-#include <type_traits>
+#include "dawn/webgpu.h"
```

Comment:
This is the Dawn version of webgpu_cpp.h. Please use the version from [emscripten-bits](https://source.chromium.org/chromium/chromium/src/+/main:out/Debug/gen/third_party/dawn/emscripten-bits/) instead.

Reply:
Oh and I think you have to copy-paste the EnumClassBitmasks in here manually, because for some reason I did that instead of including it separately.

Reply:
Though I guess we now also need to copy `webgpu_cpp_chained_struct.h` in here too so maybe we should go ahead and make `EnumClassBitmasks.h` separate as well.

Reply:
To make progress in this PR, I've reverted local changes related to EnumClassBitmmasks.